cl8-web/package.json

```diff
@@ -7,7 +7,7 @@
   "scripts": {
     "serve": "vue-cli-service serve --open",
     "build": "vue-cli-service build",
-    "test": "vue-cli-service test",
+    "test": "vue-cli-service test:unit",
```

Comment:
I believe the recommended task is `"test:unit": "vue-cli-service test:unit"`, so you can add additional tasks for e2e, etc.

Reply:
Yeah, my thinking here was to have a single test command that would trigger _all_ the tests, and fill this out when we have them.

I'm happy to make the change tho, so we can at least see the state of the unit tests before we have more kinds under CI.